environments/azure/flyte-core/locals.tf

```diff
@@ -3,9 +3,9 @@ locals {
   environment        = "deployment"
   tenant             = "mytenant"
   location           = "eastus" #Azure region or zone
-  subscription_id    = "8c8589f3-42da-4083-be83-ff9b12412edd"
-  tenant_id          = "35356303-ece0-4649-85bb-c8a9c67fd341"
-  
+  subscription_id    = "<MY_SUBSCRIPTION_ID>"
+  tenant_id          = "<MY_TENANT_ID>"
```

Comment:
Should consider wiring these up as variables with no default, that way if a user runs `terraform apply` it will prompt them for these values, or they can pass via a `.tfvars` file.